src/components/LinkWrapper.tsx

```diff
@@ -0,0 +1,4 @@
+const LinkWrapper = ({ condition, wrapper, children }) =>
```

Comment:
Can we rename it to `ConditionalLinkWrapper`?

Reply:
@Agustina-Carrion You still have this on your list?

Reply:
Yes, i'll do this now if you want to continue merging to main. It'll be done in the next minutes